lib/m3u8-parser.js

```diff
@@ -22,6 +27,26 @@ var m3u8parser = module.exports = function() {
       self.emit('item', line.trim());
     }
   }
+  function ytdlParse() {
+    var time = self.ytdlSavedLines[self.ytdlSavedLines.length - 2];
+    if (!time) {
+      self.emit('ytdl-error', new Error('Time is null/undefined.\n' + time));
+      return;
+    }
+    time = parseInt(time.replace(/#EXTINF:/g, "").replace(/,/g, ""));
+    if (self.isFirstTime) time = 1;
```

Comment:
When initially starting this loop, it will wait 5 seconds before actually outputting a stream, so we check if it's the first time so it can begin streaming immediately instead of waiting for the second packet. This will wait 1 second, to make sure the packet is parsed correctly and to give us time to live.